pkg/smartcontract/manifest/standard/comply.go

```diff
@@ -17,7 +17,8 @@ var (
 	ErrSafeMethodMismatch    = errors.New("method has wrong safe flag")
 )
 
-var checks = map[string]*manifest.Manifest{
+var checks = map[string]*Standard{
+	manifest.NEP11StandardName: nep11Base,
```

Comment:
I think we should have nep11divisible and nep11nondivisible here and check for compliance with either of these.

Reply:
Fixed.